py/src/braintrust/test_version.py

```diff
@@ -0,0 +1,8 @@
+from braintrust import version
+
+
+def test_version():
+    # Not the most interesting test, but it's a quick sanity check
+    # that the templating during the build process works.
+    assert version.VERSION
+    assert version.GIT_COMMIT
```

Comment:
could we test that these aren't equal to the pre-templating values if we're running from a wheel?

Reply:
Good catch. I actually wasn't testing the wheel. And it only took my 5 hours to figure it out.